apisix/cli/ngx_tpl.lua

```diff
@@ -677,8 +677,8 @@ http {
             if ($http_x_forwarded_for != "") {
                 set $var_x_forwarded_for "${http_x_forwarded_for}, ${realip_remote_addr}";
             }
-            if ($http_x_forwarded_port != "") {
-                set $var_x_forwarded_port $http_x_forwarded_port;
+            if ($http_x_forwarded_host != "") {
+                set $var_x_forwarded_host $http_x_forwarded_host;
```

Comment:
Do we need to add the removed directive back?

Reply:
My understanding is that this directive is already implemented in apisix.http_access_phase() and is redundant here. Is my understanding correct?  If not,   I am not sure what you mean " Some extra changes are required. See https://github.com/apache/apisix/pull/8200/files"  2 days ago.